packaging/suse/_service

```diff
@@ -0,0 +1,27 @@
+<services>
+    <service name="tar_scm" mode="disabled">
+        <param name="url">https://github.com/%%REPOSITORY%%.git</param>
+        <param name="scm">git</param>
+        <param name="revision">%%VERSION%%</param>
```

Comment:
this should not be `VERSION`, it should be `REVISION` instead
```suggestion
        <param name="revision">%%REVISION%%</param>
```

Reply:
Correct